app/views/layouts/lookup.html.erb

```diff
@@ -47,6 +47,46 @@
     });
   </script>
   <%= yield :head %>
+<script type="text/javascript" src="https://cdn.weglot.com/weglot.min.js"></script>
+<script>
+    Weglot.initialize({
+        api_key: 'wg_998f62092305ff602e8efd215c8fd0633'
```

Comment:
same here - also can this just render the partila layout/_weglot so the code isn't repeated?